modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextUndoRedo.vue

```diff
@@ -4,31 +4,24 @@
     class="apos-admin-bar__control-set apos-admin-bar__control-set--context-controls"
     name="flip"
   >
-    <!-- need a tooltip even on a disabled button -->
-    <div
-      :key="'undo'"
-      v-tooltip="undoTooltips.undo"
-    >
-      <AposButton
-        :disabled="patchesSinceLoaded.length === 0"
-        type="subtle" :modifiers="['small', 'no-motion']"
-        label="Undo" class="apos-admin-bar__context-button"
-        icon="undo-icon" :icon-only="true"
-        @click="undo"
-      />
-    </div>
-    <div
-      :key="'redo'"
-      v-tooltip="undoTooltips.redo"
-    >
-      <AposButton
-        :disabled="undone.length === 0"
-        type="subtle" :modifiers="['small', 'no-motion']"
-        label="Redo" class="apos-admin-bar__context-button"
-        icon="redo-icon" :icon-only="true"
-        @click="redo"
-      />
-    </div>
```

Comment:
unrelated to main pr
- cleans up old way to implement a tooltip on a button
- vertically aligns buttons with other elements in the bar (save indicator)